src/kiwixapp.cpp

```diff
@@ -316,10 +316,12 @@ void KiwixApp::createAction()
 
     CREATE_ACTION(NewTabAction, gt("new-tab"));
     SET_SHORTCUT(NewTabAction, QKeySequence::AddTab);
+    mpa_actions[NewTabAction]->setToolTip(gt("new-tab").append(" (Ctrl+T)"));
```

Comment:
The sequence for AddTab may not be `Ctrl+T` depending of the platform.
We must not hardcode this.
The best way is to get the string from https://doc.qt.io/qt-5/qkeysequence.html#toString
This way we will always display the right shortcut to the user.

I also wonder if the property `shortcutVisibleInContextMenu` of the action itself (https://doc.qt.io/qt-5/qaction.html#shortcutVisibleInContextMenu-prop) could help us also.